client/src/components/Ui/PinningHeader/index.js

```diff
@@ -0,0 +1,18 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+import PropTypes from "prop-types";
+
+import classes from "./styles.css";
+
+const PinningHeader = ({ children }) => {
+  return <div className={classes.component}>{children}</div>;
```

Comment:
it's webpack loader. Every imports, webpack try to resolve the extensions and passed the loader on the file https://github.com/webcompat/webcompat-metrics/pull/31/files#diff-80e4d88779d3be2308dfe08257ed3275R31

When you see: `import classes from "./styles"`
```
console.log(classes, "classes")
#{
  baseline: "src-components-Header--baseline--dzCpr",
​   button: "src-components-Header--button--3CBMC",
​   component: "src-components-Header--component--His3H",
   link: "src-components-Header--link--3Qqth",
   logo: "src-components-Header--logo--1ZE0j",
}
console.log(typeof(classes))
# object
```

It's easy to maintain, because if you remove a classe, you have a wanring and un undefinied. If you let a classe without any use, the linter is not happy, so no legacy. Plus, i added a config to hash the classname to avoid bad overide by thirdparty or any bad using of css